### ISSUE TITLE
Artifacts on the validation dataset of Nersemblev2

### ISSUE THREAD

**Issue by zhoutianyang2002** [gapszju/RGBAvatar#14]
Dear author,

Thanks for your excellent work!

I am trying to reconstruction the avatars in the NersembleV2 dataset using the latest version of RGBAvatar. I trained on the 253 ID, following the preprocess of VHAP. The results on the training set seems not bad. However, when I test on the validation set(a novel view of camera 8), I found obvious shadows on the avatar's face. I do confused about that, why the result degraded so much? I follow the default settings of RGBAvatar, only use sh0 for color. So that is very strange.

The following are the results on the validation set and training set(camera 6), respectively. Could you please give me some suggestions?

Thank you very much! Best wishes!

<img width="550" height="802" alt="Image" src="https://github.com/user-attachments/assets/3703c0e0-b425-4336-bddc-b2757658f3fa" />

<img width="550" height="802" alt="Image" src="https://github.com/user-attachments/assets/419f4e9a-39ae-41c2-80a2-823762bd8202" />
